packages/documentation-site/patternfly-docs/content/accessibility/about-modal/about-modal.md

```diff
@@ -3,26 +3,78 @@ id: About modal
+- Provide an `aria-label` to the about modal if there is no product name heading.
+- Ensure the first focusable element or the about modal itself receives focus when it is opened.
+- Ensure content behind the about modal cannot be interacted with or navigated to via mouse, keybaord, or other assistive technologies such as screen readers.
+- Ensure the element that previously had focus before the about modal was opened receives focus upon closing the about modal.
 
-<br/>
+## Testing
 
-The following props can be added or are customizable in PatternFly:
+At a minimum, an about modal should meet the following criteria:
 
-| React component| React prop | Which HTML element it appears on in markup | Explanation | 
-|---|---|---|---|
-| AboutModal | brandImageAlt | .pf-v5-c-about-modal-box__brand-image | The alternate text of the brand image |
-| AboutModal | closeButtonAriaLabel | .pf-v5-c-modal-box__close .pf-v5-c-button  | Provides an accessible name for the close button as it uses an icon instead of text. Default is “Close dialog”. Only used if you are customizing. |
+<List isPlain>
+  <ListItem>
+    <Checkbox id="aboutModal-a11y-checkbox-1" label={<span>An <code class="ws-code">aria-label</code> is passed if the about modal does not have a product name heading.</span>} description="This gives the about modal an accessible name, providing context of its purpose or content to users." />
+  </ListItem>
+  <ListItem>
+    <Checkbox id="aboutModal-a11y-checkbox-2" label="The first focusable element, or the about modal itself, receives focus when the about modal is opened." />
+  </ListItem>
+  <ListItem>
+    <Checkbox id="aboutModal-a11y-checkbox-3" label="Any content behind the about modal cannot be interacted with or navigated to." descriptjon="Upon reaching the end of the about modal, focus should either wrap back to the beginning of the about modal (for keyboard) or navigation should simply be prevented (for assistive technologies such as screen readers)." />
+  </ListItem>
+  <ListItem>
+    <Checkbox id="aboutModal-a11y-checkbox-4" label="The last element to have focus before the about modal was opened should receive focus after it is closed." descriptjon="Typically this will be whatever triggered the about modal to open. If that element no longer exists, then the element closest to it should receive focus." />
+  </ListItem>
+</List>
 
+## React customization
 
+The following React props have been provided for more fine-tuned control over accessibility.
 
+| Prop | Applied to | Reason | 
+|---|---|---|
+| `brandImageAlt="[alt text of the brand image]"` | `AboutModal` | Adds alternative text for the brand image passed to `brandImageSrc`. Typically this should either be an empty string `''` if the brand image is purely decorative, or the product name. |
```

Comment:
This looks awesome!! I noticed that we didn't call out the `brandImageAlt` though as part of the accessibility or testing sections. Was that intentional?

Reply:
Not particularly, I'll push an update that adds that in other sections